e2e/features/layers/layers-vector-test.js

```diff
@@ -30,8 +30,8 @@ module.exports = {
     c.url(c.globals.url + damsLayerWMSZoomLevelQuerystring);
     c.waitForElementVisible('#active-GRanD_Dams .fa-hand-pointer', TIME_LIMIT, () => {
       c.moveToElement('#wv-map-geographic', 400, 200)
-        .mouseButtonClick(0);
-      c.pause(200);
+        .click('#wv-map-geographic');
+      c.pause(5000);
```

Comment:
This pause is also pretty long, can we not just `waitForElementVisible` on something here?

Reply:
Reduced pause, this was left over from debugging.